### ISSUE TITLE
DST Staging Review: Change File and Delete buttons do not have unique accessible names

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#3083]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** File Input Component
**Staging Review ticket:** [Staging Review: (DS) Converted File Input Component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77552)

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**This is an issue with the:** code
**High-priority:** yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

The Change File and Delete buttons do not have unique accessible names in cases where multiple files are uploaded (either the `va-file-input-multiple` component or multiple instances of the `va-file-input` component on a single page). A screen reader user may have trouble differentiating the buttons, and may accidentally change or delete a file they intended to keep.

<details><summary>Link, screenshot or steps to recreate</summary>

![Multiple file upload component with two files uploaded. Each file listed has a Change File button and a Delete button.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/Brian-77552-1.png?raw=true)

Change file buttons are both coded as:
```
<button class="usa-button" aria-label="Change file" type="button" part="button"><va-icon class="hydrated"></va-icon>Change File</button>
```
And similar for Delete.

</details>

## Recommended action
Recommend adding the filename to the `aria-label`, eg. `aria-label="Change file [filename.abc]"`.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/3083?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @powellkerry @rmessina1010 @rsmithadhoc
